src/axolotl/utils/models.py

```diff
@@ -52,6 +56,20 @@ def check_model_config(cfg: DictDefault, model_config: AutoConfig):
             "Please use the `gptq` flag to train quantized model or point to a non-quantized model."
         )
 
+    lora_modules_to_save = get_linear_embedding_layers(model_config.model_type)
+    if (
+        cfg.adapter
+        and cfg.tokens
+        and (
+            not cfg.lora_modules_to_save
+            or not all(x in cfg.lora_modules_to_save for x in lora_modules_to_save)
+        )
+    ):
+        lora_modules_to_save = ", ".join(map(lambda x: f"`{x}`", lora_modules_to_save))
```

Comment:
```suggestion
        lora_modules_to_save = ", ".join([f"`{x}`" for x in lora_modules_to_save])
```

Could this be refactored to so?